site/guide/5-responses.md

```diff
@@ -237,15 +237,15 @@ async fn files(file: PathBuf) -> Option<NamedFile> {
 
 ### `Result`
 
-`Result` is a special kind of wrapping responder: its functionality depends on
-whether the error type `E` implements `Responder`.
+`Result` is a _wrapping_ responder: a `Result<T, E>` can only be returned when
```

Comment:
Since the previous paragraph started identically to this, can we stick an "also" or "another" early on in the sentence?

Reply:
Sure!